schemas/stsci.edu/asdf/transform/label_mapper-0.1.0.yaml

```diff
@@ -0,0 +1,138 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/transform/label_mapper-0.1.0"
+tag: "tag:stsci.edu:asdf/transform/label_mapper-0.1.0"
+title: >
+  Represents a _LabelMapper.
```

Comment:
The Python class `_LabelMapper` is just an implementation detail.  Maybe say something like:  "Represents a mapping from a coordinate value to a label."